tests/integration/test_data_cli.py

```diff
@@ -725,8 +732,49 @@ def test_search_delete_nonexistant_search_id(invoke, search_id, search_result):
     assert result.exit_code == 1
 
 
+@pytest.mark.parametrize("item_types",
+                         ['PSScene', 'SkySatScene', 'PSScene, SkySatScene'])
+@respx.mock
+def test_search_update_success(invoke,
+                               search_id,
+                               search_result,
+                               item_types,
+                               search_filter):
+    update_url = f'{TEST_SEARCHES_URL}/{search_id}'
+    mock_resp = httpx.Response(HTTPStatus.OK, json=search_result)
+    respx.put(update_url).return_value = mock_resp
+
+    name = "search_name"
+
+    result = invoke([
+        'search-update',
+        search_id,
+        name,
+        item_types,
+        json.dumps(search_filter)
+    ])
+
+    assert not result.exception
```

Comment:
@kevinlacaille maybe make a simple assertion on the value of `result.output`? Like a check that the search name gets propagated into the output.